### PR TITLE
Assignment 1: Jacob Newcomb & Jessica Ho

### PR DESCRIPTION
**Pull request by jacobnewcomb**
Pull Request title includes "Assignment 1" in the title ✔
Pull Request targets Assignment 1 branch ✔
Issue 1: Application no longer crashes ✔
Issue 2: Unit test properly passes ✔
@JessicaH46

### CODE REVIEW

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 1]
```diff
@@ -24,7 +24,7 @@ public static void Main(string[] args)
 
         public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
         {
-            return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+            return ((double)numberCorrectAnswers / (double)numberOfQuestions * 100) + "%";
```
Comment (FenrisStargazer) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:27]: This technique didn't seem to work (as far as making the unit tests pass) when I tried it at first, so I'm curious what was different.

**Review by FenrisStargazer — COMMENTED** [review]
Pull Request title includes "Assignment 1" in the title ✔
Pull Request targets Assignment 1 branch ✔
Issue 1: Application no longer crashes ✔
Issue 2: Unit test properly passes ✔

Extra Credit
Come up with a new feature for the application
Issue 3: Add a new feature request implemented ❌
Issue 3: Feature request unit tested ❌

I think you did good making the edits very concise, minimizing changes to the code while fulfilling the requirements of making it run and perform adequately in the unit tests. I'd be interested to see what ideas you could come up with for the EC of adding an additional feature though.

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs (hunk 1 above)
Comment (jacobnewcomb) [reply PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:27]: I'm not sure what that could be about! From what I looked up on doing double division in C#, this was the method that popped up.

Comment (FenrisStargazer) [reply PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:27]: Yeah when I tried it again in a fresh copy of the code it seemed to work just fine. I think I must've typed something wrong or made another change that affected it.

**Pull request closed without merging**